packages/experiments/src/components/Tile/examples/Tile.Media.Example.tsx

```diff
@@ -146,6 +150,8 @@ const ImageTile: React.FunctionComponent<IImageTileProps> = (props: IImageTilePr
 export interface ITileMediaExampleState {
   imagesLoaded: boolean;
   nameplateOnlyOnHover: boolean;
+  generateLinks: boolean;
+  linkInNewTab: boolean;
```

Comment:
might be missing something here but there should be logic to actually open the link in a new tab somewhere right?

Reply:
Yes, it's on line 173, inside the render() method. Certainly we could move that part into the state if you think that's more elegant, but the way I did it seems consistent with the existing design (the other state parameters were both booleans). The demo works as intended on my machine in both Edge and Chrome.

Reply:
This is fine, thanks